src/handlers/localstorage/accountLocal.ts

```diff
@@ -229,6 +232,36 @@ export const saveLocalTransactions = (
     transactionsVersion
   );
 
+/**
+ * @desc get ENS registrations
+ * @param  {String}   [address]
+ * @param  {String}   [network]
+ * @return {Object}
+ */
+export const getLocalENSRegistrations = (
+  accountAddress: any,
+  network: any
+): Promise<ENSRegistrations> =>
+  getAccountLocal(ENS_REGISTRATIONS, accountAddress, network);
```

Comment:
the `getAccountLocal` function has a 4th param which represents the empty state to return; by default it is `[]` which is likely not the expected empty state you are expecting

Reply:
oh nice